.github/workflows/doc-build.yml

```diff
@@ -80,7 +80,10 @@ jobs:
         # incremental docbuild may introduce broken links (inter-file references) though build succeeds
         run: |
           set -ex
-          make doc-clean doc-uninstall sagelib-clean && git clean -fx src/sage && ./config.status && make doc-html
+          mv /sage/local/share/doc/sage/html/en/.git /sage/.git-doc
+          make doc-clean doc-uninstall sagelib-clean && git clean -fx src/sage
+          mkdir -p /sage/local/share/doc/sage/html/en/ && mv /sage/.git-doc /sage/local/share/doc/sage/html/en/
+          ./config.status && make doc-html
```

Comment:
Move `.git-doc` to `.git-doc`? No `.git`?

Where is the code to generate `CHANGES.html`? No idea how this works.

Reply:
Thanks, fixed now.

`CHANGES.html` is generated in the "Copy docs" step

Reply:
Are you aware that L53 is very similar to L100 in the "Copy docs" step. This seems to be a mistake to me...

Reply:
Yes, you are.

Reply:
What is the net effect of L83 and L85. It is moving `.git` directory to and from `.gitdoc`. Why?

Reply:
In particular, why `mkdir -p /sage/local/share/doc/sage/html/en/`. The directory already exists. Why create it again?

Reply:
I'm moving the `.git` directory away and back to protect it from `make doc-uninstall`

Reply:
OK.